lib/default_questions_helper.rb

```diff
@@ -33,12 +33,13 @@ def ask_base_questions(options, manifest_hash = { api: {}, dependency_repository
       ask_for_dependency_repository(manifest_hash)
     end
 
-    manifest_hash[:ui_builder_support] = agree "[?] Should this plugin be available in Zapp UI Builder? (Y/n)"
-    package_name = Question.ask_base("Package name:")
+    manifest_hash[:ui_builder_support] = agree "[?] Should this plugin be available in Zapp UI Builder apps? (Y/n)"
+    
+    package_name = Question.ask_base("Dependency name - The name of the Pod or Maven(Bintry) of your plugin:")
```

Comment:
I would remove Bintray, its not necessarily that. I would also follow the same styling - `Dependency name (name of the Pod or Maven library of you plugin):`